tests/tck/features/update/Update.feature

```diff
@@ -1207,3 +1207,135 @@ Feature: Update string vid of vertex and edge
       | like.likeness | like.new_field |
       | 1.0           | "111"          |
     Then drop the used space
+
+  Scenario: Update player(name string, age int, hobby List< string >, ids List< int >, score List< float >)
+    Given an empty graph
+    And create a space with following options:
+      | partition_num  | 1                |
+      | replica_factor | 1                |
+      | vid_type       | FIXED_STRING(20) |
+    And having executed:
+      """
+      CREATE TAG player(name string, age int, hobby List< string >, ids List< int >, score List< float >);
+      """
+    And wait 3 seconds  # Wait for the index to be created
+    # Insert vertex data for player100
+    When try to execute query:
+      """
+      INSERT VERTEX player(name, age, hobby, ids, score) VALUES "player100":(
+          "Tim Duncan", 42, ["Basketball", "Swimming"], [100, 528], [50.0, 22.0]
```

Comment:
Is there testcase cover some corner cases, e.g.
1. empty list/set, insert or update.
2. Fire some error conditions: for example: 
   - type mismatched, like insert List[?] to Set[?] , or insert List[int] to List[string];
   - invoke new added function with wrong type, e.g. add string to List[int] 
   - what happends if try to create index on list/set column?
3. very large list/set, according to your implementation, there is seem no limit (or more accurate max of int32) on the size of list or set, right? Should we support that large size list/set?

Reply:
1. I have added test cases: Insert and update empty list/set
2. If the inserted data type is inconsistent, an error will be reported: Storage Error: The data type does not meet the requirements. Use the correct type of data.
    I also restricted the creation of list/set indexes and added test cases.
3. I have added a restriction on the array length